code/datums/ai/_ai_controller.dm

```diff
@@ -24,19 +25,34 @@ have ways of interacting with a specific atom and control it. They posses a blac
 	var/continue_processing_when_client = FALSE
 	///distance to give up on target
 	var/max_target_distance = 14
+	///Cooldown for new plans, to prevent AI from going nuts if it can't think of new plans and looping on end
+	COOLDOWN_DECLARE(failed_planning_cooldown)
+	///All subtrees this AI has available, will run them in order, so make sure they're in the order you want them to run
+	var/list/planning_subtrees = list()
+
+	///Movement related things here
 	///Reference to the movement datum we use. Is a type on initialize but becomes a ref afterwards.
 	var/datum/ai_movement/ai_movement = /datum/ai_movement/dumb
 	///Cooldown until next movement
 	COOLDOWN_DECLARE(movement_cooldown)
 	///Delay between movements. This is on the controller so we can keep the movement datum singleton
 	var/movement_delay = 0.1 SECONDS
+
+	/////The variables below are fucking stupid and should be put into the blackboard at some point.
```

Comment:
```suggestion
	// The variables below are fucking stupid and should be put into the blackboard at some point.
```

Otherwise, you're going to get a doc comment for something that isn't documentation.